astropy_helpers/sphinx/ext/automodapi.py

```diff
@@ -52,6 +52,14 @@
         given, only objects that are actually in a subpackage of the package
         currently being documented are included.
 
+    * ``:inherited-members: True | False``
+        The global sphinx configuration option
+        ``automodsumm_inherited_members`` decides if members that a class
+        inherits from a base class are included in the generated
+        documentation. The option ``:inherited-members:`` allows to
+        overrride this global setting.
```

Comment:
nitpick: "allows to overrride" -> "allows overriding" or "allows the user to override" or similar.